pkg/util/test/util.go

```diff
@@ -288,6 +288,42 @@ func ValidateStorageCluster(
 
 	pxEndpoint = fmt.Sprintf("%s:%d", svc.Spec.ClusterIP, servicePort)
 
+	if err := validatePodsByName(cluster.Namespace, "portworx-pvc-controller", timeout, interval); err != nil {
+		return err
+	}
+
+	if cluster.Spec.Stork != nil && cluster.Spec.Stork.Enabled {
+		if err := validatePodsByName(cluster.Namespace, "stork", timeout, interval); err != nil {
+			return err
+		}
+		if err := validatePodsByName(cluster.Namespace, "stork-scheduler", timeout, interval); err != nil {
+			return err
+		}
+	}
+
+	if cluster.Spec.Autopilot != nil && cluster.Spec.Autopilot.Enabled {
```

Comment:
Should also validate image, see example from my PR https://github.com/portworx/spawn/pull/518
`		// Autopilot
		if spec.Spec.Autopilot != nil {
			params.Autopilot = spec.Spec.Autopilot.Enabled
			if spec.Spec.Autopilot.Image != "" {
				params.AutopilotImage = spec.Spec.Autopilot.Image
			}
		}`